packages/core/src/CoreAPI.ts

```diff
@@ -0,0 +1,88 @@
+import { IdenityRegistry } from '@tangleid/did';
+import { signRsaSignature, verifyRsaSignature } from '@tangleid/jsonld';
+import { DocumentLoader } from './jsonld/documentLoader';
+
+import { IriProvider, Seed, Did, PublicKeyMeta, PrivateKeyPem, PublicKeyPem } from '../../types';
+
+export type Settings = {
+  provider?: IriProvider;
+};
+
+export class CoreAPI {
+  private registry: IdenityRegistry;
+  private documentLoader: DocumentLoader;
+
+  constructor(settings: Partial<Settings> = {}) {
+    this.registry = new IdenityRegistry({ provider: settings.provider });
+    this.documentLoader = new DocumentLoader(this.registry);
+  }
+
+  /**
+   * Publish the DID document to the Tangle MAM channel with specific network.
```

Comment:
Change from "Tangle MAM channel with specific network" to "MAM channel with specific Tangle network". Please be aware of the fact of second layer for MAM channel.

Reply:
Fixed.